src/js/tabs/receiver.js

```diff
@@ -24,28 +24,28 @@ const receiver = {
     rateChartHeight: 117,
     analyticsChanges: {},
     needReboot: false,
-    elrsPassphraseEnabled: false,
+    elrsBindingPhraseEnabled: false,
 };
 
 receiver.initialize = function (callback) {
     const tab = this;
 
     GUI.active_tab = 'receiver';
 
-    function lookup_elrs_passphrase(uidString) {
-        const passphraseMap = getConfig('passphrase_map').passphrase_map || {};
+    function lookup_elrs_binding_phrase(uidString) {
```

Comment:
If you add it maybe also change to camel case

Reply:
Done in 3fcf81d67faa972c6e91c4547b456546c5623c09